src/lib/utils/legendary.ts

```
@@ -0,0 +1,153 @@
+    ownedApps.update(current => {
+      const existingGame = current.find(g => g.id === appId);
+      if (existingGame) {
+        existingGame.installed = false;
+      }
+
+      return current;
+    });
+
+    return data;
+  }
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)